types/react-bootstrap-table-next/index.d.ts

```diff
@@ -589,10 +591,10 @@ export type TableColumnFilterProps<FT = any, T extends object = any> = Partial<{
     /*
      * export filter function to allow users to access filter method externally.
      */
-    getFilter: (filter: FT) => void;
+    getFilter: (filter: (value: FV) => void | T[]) => void;
```

Comment:
Reason: Filter is a function
See https://github.com/Avasam/speedrun.com_global_scoreboard_webapp/blob/575459d412c41518925a2ec550b619f8d0c7ac05/global-scoreboard/src/Components/GameSearch/GameSearch.tsx#L34